configuration/configure-point-for-storage-management.md

```diff
@@ -64,4 +64,4 @@ After completing the above, restart **both** App Services inside the Point resou
 
 ## Related Articles
 
-* [Learn How to Optimize Storage Using Syskit Point](../governance-and-automation/optimize-storage.md)
\ No newline at end of file
+* [Learn how to optimize Storage Management using Syskit Point](../reporting/storage-management/storage-management.md)
```

Comment:
ovo mi čudno zvuči
može biti ili 'Learn how to optimize storage using Syskit Point' ili 'Learn how to manage storage using Syskit Point'